runhouse/main.py

```diff
@@ -390,13 +390,14 @@ def cluster_up(
 
     Example:
 
-        ``$ runhouse cluster up /sashab/rh-basic-cpu``
+        ``$ runhouse cluster up rh-basic-cpu``
 
     """
     current_cluster = rh.cluster(name=cluster_name, dryrun=True)
 
     try:
         current_cluster.up_if_not()
+        console.print("Cluster is already up.")
```

Comment:
How about just `"Cluster is already up."`? Because we might call `runhouse cluster up` on a terminated cluster, and that it will be a bit wired to see` "Cluster is already up"` printed to the console in this case.

Reply:
oh we prob shouldn't do that, we should be able to know if the cluster is up or not, and then print a message if its already up.. ill change that